### PR TITLE
[DROOLS-3487] Initial import

### PR DESCRIPTION
**Pull request by gitgabrio**
@mbiarnes @mareknovotny @baldimir @ederign @porcelli @manstis @danielezonca

See https://issues.jboss.org/browse/DROOLS-3847

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+  <parent>
+    <groupId>org.jboss</groupId>
+    <artifactId>jboss-parent</artifactId>
+    <version>30</version>
+  </parent>
+
+  <groupId>org.kie</groupId>
+  <artifactId>kie-gwthelper-maven-plugin</artifactId>
+  <packaging>maven-plugin</packaging>
+  <version>${version.org.kie.gwthelper.maven}</version>
```
Comment (baldimir) [inline pom.xml]: Is this still needed? I mean to have this version in a property.

**Review by baldimir — COMMENTED** [review]
Please see my comment about the version. Other than that, please consider adding tests to this project, so the plugin is unit-tested. You can find some information how to do that here [1]. But that could be done later.  

[1] https://maven.apache.org/plugin-developers/plugin-testing.html

**Review by danielezonca — APPROVED** [review]
Fine for me

File: pom.xml (hunk 1 above)
Comment (mareknovotny) [reply pom.xml]: it should not be needed for a simple project

**Review by mareknovotny — APPROVED** [review]
looks good to me

**Comment by gitgabrio** [comment]
> Please see my comment about the version. Other than that, please consider adding tests to this project, so the plugin is unit-tested. You can find some information how to do that here [1]. But that could be done later.
> 
> [1] https://maven.apache.org/plugin-developers/plugin-testing.html
Hi Tibor,
thanks, I already have some experience with testing maven plugins. I'll create a new ticket to add some.

**Review by baldimir — APPROVED** [review]

**Review by manstis — APPROVED** [review]

**Comment by gitgabrio** [comment]
@mbiarnes @mareknovotny @baldimir @ederign @porcelli @manstis @danielezonca
I'm merging this one since it is basically just a move from one place to another one.
Thanks for the review.
Gabriele

**Pull request merged**